src/libraries/System.Text.RegularExpressions/System.Text.RegularExpressions.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
```

Comment:
Did you edit this file manually (and by manually I meant with VS)? If so we probably want to instead use slngen instead so as to prevent future conflicts when it is run again on the repo.

Reply:
If you want to run slngen locally, just run the following from the root of the repo:
```
dotnet build .\src\libraries\slngen.proj /p:SolutionNameOverride=System.Text.RegularExpressions
```

Reply:
Thanks.